System/ContentFile.cpp

```diff
@@ -1,724 +1,414 @@
-//////////////////////////////////////////////////////////////////////////////////////////
-// File:            ContentFile.cpp
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Source file for the ContentFile class.
-// Project:         Retro Terrain Engine
-// Author(s):       Daniel Tabar
-//                  data@datarealms.com
-//                  http://www.datarealms.com
-
-
-//////////////////////////////////////////////////////////////////////////////////////////
-// Inclusions of header files
-
 #include "ContentFile.h"
```

Comment:
I will comment on ContentFile.cpp separately, time has defeated me for now but I want to get this review up for you.